game/scripts/npc/items/custom/item_sonic.txt

```diff
@@ -0,0 +1,92 @@
+"DOTAItems"
+{
+  //=================================================================================================================
+  // Recipe: Sonic
+  //=================================================================================================================
+  "item_recipe_sonic"
+  {
+    "ID"                  "8473"
```

Comment:
You'll need to update these ID's since lots of new stuff made it in, such as electrician.